### PR TITLE
design fixes

### LINKED ISSUE #21: Design fixes
# REQUIREMENTS
Make some design fixes to improve the user experience:
-Remove the keyboard so that it does not cover the log in input.
-Add an alert after submitting the test results.

# DESIGN
Provide references to images of the design, how it looks like now, and how it should look like.
Captura de Pantalla 2021-04-28 a la(s) 11 52 10
Captura de Pantalla 2021-04-28 a la(s) 12 23 44

# Acceptance Criteria
List all required acceptance criteria
- [ ] Improve UI/UX
- [ ] Good quality code

### PR DESCRIPTION
**Pull request by genesis9432**
# Description
-We make some design fixes in the log in screen, we remove the keyboard so that it does not cover the email and password input, also added an alert after submitting the test results in the test results screen to improve the UX.

 # Testing
 -git clone repository
-git checkout feature#21/DesignFixes
-npm install

 
  ## Screenshots 
Captura de Pantalla 2021-04-28 a la(s) 11 52 10
Captura de Pantalla 2021-04-28 a la(s) 12 54 10

### CODE REVIEW

File: AlertaCovid/src/components/atoms/ResultAtoms/BtnResult.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+    );
+  };
+
+
+
   return (
     <TouchableOpacity
       style={ResultStyles.btnContainerSubmit}
-      onPress={() => checkTest(testResult, test_positive)}>
+      onPress={()=> Alerta(checkTest(testResult, test_positive))}>
```
Comment (fernandoamz) [inline AlertaCovid/src/components/atoms/ResultAtoms/BtnResult.js]: change all the spanish names to english

File: AlertaCovid/src/components/atoms/ResultAtoms/BtnResult.js [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
 import {test_positive} from '../../../redux/actions/HealthActions';
 import {connect} from 'react-redux';
 import {checkTest} from '../../../utils/functions';
-const Submit = ({testResult, test_positive}) => {
+const Submit = ({testResult, test_positive, navigation}) => {
+
+  const Alerta = () => {
+    Alert.alert(
+      'Submited successfully',
+      'Your data has been submited successfully',
```
Comment (fernandoamz) [inline AlertaCovid/src/components/atoms/ResultAtoms/BtnResult.js]: Add this to constants file

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by LeyaimMagma — DISMISSED** [review]

File: AlertaCovid/src/components/atoms/ResultAtoms/BtnResult.js [hunk 3]
```diff
@@ -1,15 +1,28 @@
...
+      ],
+      {cancelable: true},
+    );
+  };
+
   return (
     <TouchableOpacity
       style={ResultStyles.btnContainerSubmit}
-      onPress={() => checkTest(testResult, test_positive)}>
+      onPress={() => Alert(checkTest(testResult, test_positive))}>
```
Comment (fernandoamz) [inline AlertaCovid/src/components/atoms/ResultAtoms/BtnResult.js:25]: You are using a reserved word to name the funtion, this is not the best practice to name a variable I recommend you change the name.

**Review by fernandoamz — APPROVED** [review]

**Review by HRuvalcabaP — APPROVED** [review]
Good job

**Review by mariam-rivera — APPROVED** [review]
Pretty nice!

**Pull request merged**